pkg/auth/audit/audit.go

```diff
@@ -208,3 +212,56 @@ func isExist(array []string, key string) bool {
 	}
 	return false
 }
+
+func (a *auditLog) concealSensitiveData(requestURI string, body []byte) []byte {
+	var m map[string]interface{}
+	if err := json.Unmarshal(body, &m); err != nil {
+		return body
+	}
+
+	var changed bool
+	if strings.HasSuffix(requestURI, "secrets") {
```

Comment:
This seems fragile if QSP gets added, or someone adds a trailing slash. Grabbing any URI with secret in it makes more sense to me. Worst case we grab too much, but its not a big performance hit.

Reply:
Switched to `strings.Contains`